docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contributing
+The `Polkadot SDK` project is an **OPENISH Open Source Project
+## What?
+Individuals making significant and valuable contributions are given commit-access to the project. Contributions are done via pull-requests and need to be approved by the maintainers.
+## Rules
+There are a few basic ground-rules for contributors (including the maintainer(s) of the project):
+1. **No `--force` pushes** or modifying the master branch history in any way. If you need to rebase, ensure you do it in your own repo. No rewriting of the history after the code has been shared (e.g. through a Pull-Request).
+2. **Non-master branches**, prefixed with a short name moniker (e.g. `gav-my-feature`) must be used for ongoing work.
```

Comment:
I think it's neater if we use the slash rather than hyphen:
```suggestion
2. **Non-master branches**, prefixed with a short name moniker (e.g. `gav/my-feature`) must be used for ongoing work.
```

Reply:
By using `/` instead of `-` it becomes ambiguous in the presence of multiple removes. Git assumes that everything before the `/` is the name of the remore.